src/classes/Standardizer/plugins/Google/getters/getChats.ts

```diff
@@ -6,7 +6,7 @@ const ACCOUNT_CHAT_FILE = 'Takeout/Hangouts/Hangouts.json'
 Google.prototype.getChats = async function getChats(options) {
   const rawChats = await this.parser.parseAsJSON(ACCOUNT_CHAT_FILE, options?.parsingOptions)
   const chats: Array<Chat> = await Promise.all(rawChats.conversations.map((conv: any) => ({
```

Comment:
```suggestion
  const chats: Array<Chat> = await Promise.all(rawChats.conversations.map((conv: any): Chat => ({
```
Use TS typing like that to avoid this kind of errors